libs/components/help-inline/src/lib/modules/help-inline/help-inline.component.ts

```diff
@@ -117,27 +107,40 @@ export class SkyHelpInlineComponent {
   @Output()
   public actionClick = new EventEmitter<void>();
 
-  protected isPopoverOpened: boolean | undefined;
-  protected popoverId: string | undefined;
-  protected popoverTemplate: TemplateRef<unknown> | undefined;
+  protected readonly defaultAriaLabel: Signal<string | undefined>;
+  protected readonly labelTextResolved: Signal<string | undefined>;
 
-  protected readonly helpSvc = inject(SkyHelpService, { optional: true });
+  readonly #format = inject(SkyAppFormat);
+  readonly #labelText = signal<string | undefined>(undefined);
+  readonly #resourcesSvc = inject(SkyLibResourcesService);
 
-  #_popoverContent: string | TemplateRef<unknown> | undefined;
+  constructor() {
+    this.defaultAriaLabel = toSignal(
```

Comment:
I would take this out of the constructor and assign it where the field is declared. I've been injecting my private services at the top so I can reference them in subsequent field declarations.